scipy/stats/tests/test_discrete_distns.py

```diff
@@ -141,6 +143,13 @@ def test_betabinom_a_and_b_unity():
     expected = np.repeat(1 / (n + 1), n + 1)
     assert_almost_equal(p, expected)
 
+@pytest.mark.parametrize('dtypes', itertools.product(*[(int, float)]*3))
```

Comment:
Oops, last thing: PEP8 requires two empty lines between top level function definitions.
```suggestion

@pytest.mark.parametrize('dtypes', itertools.product(*[(int, float)]*3))
```